metals/src/main/scala/scala/meta/internal/metals/CodeLensProvider.scala

```diff
@@ -61,7 +89,16 @@ final class DebugCodeLensProvider(
               .get(symbol)
               .map(testCommand(target, _))
               .getOrElse(Nil)
-            main ++ tests
+            val docWithPath = TextDocumentWithPath(textDocument, path)
+            val gotoSuperMethod =
+              createSuperMethodCommand(
+                docWithPath,
+                symbol,
+                occurrence.role,
+                emptyLensGoSuperCache(),
+                search
+              )
+            main ++ tests ++ gotoSuperMethod
```

Comment:
I was thinking of having it calculated with a method like `SemanticDBDoc => CodeLenses`, which could be in a separate trait. Both implementations would in a list and we could join their results.